handbook/people-ops/hiring.md

```diff
@@ -17,3 +17,15 @@ We use [Google Hire](https://hire.withgoogle.com/t/sourcegraphcom/hiring/landing
    - Interviewers take written notes during the interview so there is an objective record of the interview that can be referenced when making a hiring decision.
    - All interviewers submit written feedback independently before being able to see feedback from other interviewers.
    - We maintain a consistent hiring decision process for all candidates.
+
+## Sharing feedback
+
+We don't share specific reasons or feedback if we decide not to move forward in our hiring process with a candidate.
+
+We can only interview and hire a finite number of people in a given amount of time, so we move forward with only a few of the best candidates at each stage of our hiring pipeline.
+
+Ultimately, each hiring decision is a complex judgement call based on many inputs.
```

Comment:
```suggestion
Ultimately, each hiring decision is a complex one based on many inputs.
```
"judgement call" to me implies lack of decision structure/framework (even though there is always a bit of judgement)